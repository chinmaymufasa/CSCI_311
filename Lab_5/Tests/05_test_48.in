2
130
5
8
2
178
5
8
2
196
5
8
2
140
5
8
2
6
5
8
-1
