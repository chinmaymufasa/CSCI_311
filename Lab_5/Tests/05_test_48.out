130 
130
130 178 
178
130 178 196 
196
130 140 178 196 
196
6 130 140 178 196 
196
